src/shared/proto/tbots_robot_msg.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto2";
+
+import "shared/proto/robot_component_status.proto";
+
+message TbotsRobotMsg
+{
+    required uint32 robot_id                           = 1;
+    repeated ErrorCode error_code                      = 2;
+    required BreakBeamStatus break_beam_status         = 3;
+    required ChipperKickerStatus chipper_kicker_status = 4;
+    required EncoderData encoder_data                  = 5;
```

Comment:
There's `EncoderData` here at the top level as well as within `MotorStatus`, which I think is confusing. I think my preference is to just have it at the top level, so it's not coupled to the `MotorStatus`